apps/finance/app/src/components/Transfers.js

```diff
@@ -227,17 +245,19 @@ const Transfers = React.memo(({ tokens, transactions }) => {
             >
               Transfers
             </div>
-            <div css="text-align: right;">
-              <Button onClick={handleDownload}>
-                <IconExternal
-                  css={`
-                    margin-right: ${1.5 * GU}px;
-                    color: ${theme.surfaceIcon};
-                  `}
-                />{' '}
-                Export
-              </Button>
-            </div>
+            {transactions.length > 0 && (
+              <div css="text-align: right;">
+                <Button onClick={handleDownload}>
+                  <IconExternal
+                    css={`
+                      margin-right: ${1.5 * GU}px;
+                      color: ${theme.surfaceIcon};
+                    `}
+                  />{' '}
+                  Export
+                </Button>
+              </div>
```

Comment:
We could refresh this part a bit since we are here:

```suggestion
              <div>
                <Button
                  icon={<IconExternal />}
                  label="Export"
                  onClick={handleDownload}
                />
              </div>
```